### DISCUSSION TITLE
Feature request: Allow support of both packr and fs

### DISCUSSION THREAD

**Discussion by Yaska1706** [gobuffalo/buffalo#2220] [Ideas]
### Description

From `v0.8.0` buffalo started using `io/fs` from the standard library which replaced `packr` and this limits backward compatibility when upgrading other buffalo versions to the latest version.

It will be convenient to allow both support for `packr` and `fs`. Instead of doing a code rewrite which can break the system.

Would love to get feedback if it is a good approach.

**Comment by paganotoni** [comment]
Hey @Yaska1706. Thanks for sharing the idea. Maintaining 2 ways of embedding files within the Buffalo library would be too much effort for the maintainers. We moved to use the `embed` package because the Go team is maintaining it and can guarantee the stability of it.

One alternative to support Packr is to write a packer FileSystem. In that case Buffalo would not need to change at all but rather the wrapper interface would allow use `Packr` Boxes for those applications that cannot move away from Packr.
